lwgenerate/azure/test-data/activity-log-with-all-subscriptions.tf

```diff
@@ -22,7 +22,7 @@ module "az_ad_application" {
 
 module "az_activity_log" {
   source                      = "lacework/activity-log/azure"
-  version                     = "~> 2.0"
+  version                     = "~> 2.7"
```

Comment:
Have we updated these versions correctly?

https://g.codefresh.io/build/649c04a31af33f4ceb64e0eb?step=integration&tab=output&logs=text